salesforce-composite/2.8/modules/ROOT/pages/index.adoc

```diff
@@ -26,7 +26,7 @@ To use this connector, you must be familiar with:
 
 Before creating an app, you must have:
 
-* Credentials to access the Salesforce Composite target resource
+* Credentials to access the Salesforce Composite Connector target resource
```

Comment:
It's actually the Salesforce Composite target resource (they are Salesforce Composite APIs)